crates/holochain/src/core/workflow/app_validation_workflow/tests.rs

```diff
@@ -193,6 +186,43 @@ async fn main_loop_app_validation_workflow() {
     )
     .await
     .unwrap();
+    assert_matches!(
+        outcome_summary,
+        OutcomeSummary {
+            ops_to_validate: 0,
+            validated: 0,
+            accepted: 0,
+            rejected: 0,
+            missing: 0,
+        }
+    );
+
+    // remove op with missing hash from validation dependency filter
+    validation_dependencies.lock().hashes_missing_for_op.clear();
```

Comment:
slightly silly now because the hash would be removed after fetching the op, but manually removing it is safer than waiting an interval for the other thread to do that